engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkModelJar.scala

```diff
@@ -34,14 +36,33 @@ class FlinkModelJar extends LazyLogging {
     tempFile
   }
 
+  private def additionalArtifactsToIncludeInJar(): List[URL] = {
+    // Including dropwizard metrics library implicitly to preserve backward compatibility
+    // with flink-metrics-dropwizard library bundled into flinkExecutor
+    val additionalJarFiles = List(
+      "flink-additional-deps/flink-metrics-dropwizard.jar",
+      "flink-additional-deps/dropwizard-metrics-core.jar"
+    )
+    val additionalJarUrls = additionalJarFiles
+      .map(filename => new File(filename))
+      .filter(_.exists())
+      .map(_.toURI.toURL)
+    additionalJarUrls
+  }
+
   private def prepareModelFile(modelData: BaseModelData): File = {
     val tempFile = generateModelFileName()
-    modelData.modelClassLoaderUrls match {
+    val implicitlyIncludedArtifacts = additionalArtifactsToIncludeInJar().toSet -- modelData.modelClassLoaderUrls.toSet
```

Comment:
How you will override this in ververica DM? shouldn't be this if extracted into some protected methods or to some method's argument?

Reply:
1. I copied whole `FlinkModelJar` into ververica module. At first, I did it because of some custom adaptations to load scala 2.12 jars. It's not really needed now. However, I think it makes some sense to keep it separated from nussknacker core codebase.
2. It would be probably good to add such flag and add `dropwizard` word to directory name. It could help in case someone wanted to disable implicit loading. However, IMO there is no way somebody would use their own metrics library. 
3. ok